launch/Mako_G-319.launch

```diff
@@ -0,0 +1,23 @@
+<launch>
+  <arg name="name" default="camera" doc="The name of the camera"/>
+  <arg name="frame_id" default="camera" doc="The frame id of the camera"/>
+  <arg name="ip" default="" doc="The IP for the camera to connect to"/>
```

Comment:
Should we fill these out with actual default IP address G-319 unit comes with?

Reply:
Good point! forgot they come with default IP addresses. I'll update.

Reply:
Turns out it's just part of the subnet that is default from factory, but the full IP is not identical.